app/frontend/src/styles/application/jobseekers/subscription.scss

```diff
@@ -0,0 +1,34 @@
+.js-enabled {
+  .subscription {
```

Comment:
could you call this `subscription-card` or something a bit more descriptive? subscription is a little generic to me and could mean a few things